Transformations/TransformEigen/TransformEigen.cpp

```diff
@@ -0,0 +1,250 @@
+/*
+Convenience functions for pose and co-ordinate system transformations.
+*/
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+#include <Eigen/Geometry>
+
+#include <opencv2/core/core.hpp>
+
+#include <cmath>
+
+#include "TransformEigen.h"
+#include <iostream>
+
+namespace TransformUtils
+{
+    enum Rotation
```

Comment:
Do you use this somewhere? I couldn't find it.

Reply:
I wanted to use it for rotmat2euler, but not sure if it's straightforward with Eigen. It can be used (as discussed above) in the non-Eigen version.